includes/utilities.inc

```diff
@@ -296,6 +294,9 @@ function islandora_multi_importer_retrieve_remote_file($url, $destination = NULL
         $path = $destination;
       }
     }
+    if(!$path) {
```

Comment:
How are we handling path here? maybe the default used outside the elvis operator when invoking this function could go here instead of returning false? that way this function always returns a valid path

Reply:
I don't think it should always return a valid path. If _imi_build_file_storage_uri() or _imi_build_file_storage_directory() aren't successful, they signal that by returning FALSE. This informs the calling function to take appropriate action. We can't assume that these functions will always be successful. That's how I see it, anyway!